### PR TITLE
fix: build break for Docker test environments

### PR DESCRIPTION
**Pull request by snadrus**
## Related Issues

Specialty builds were not automatically ran at PR time, so were caught after Submit. 

## Checklist

Before you mark the PR ready for review, please make sure that:

- [ ] Commits have a clear commit message.
- [ ] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] Update CHANGELOG.md or signal that this change does not need it.
  - If the PR affects users (e.g., new feature, bug fix, system requirements change), update the CHANGELOG.md and add details to the UNRELEASED section.
  - If the change does not require a CHANGELOG.md entry, do one of the following:
    - Add `[skip changelog]` to the PR title
    - Add the label `skip/changelog` to the PR
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: build/buildconstants/params_interop.go [hunk 1]
```diff
@@ -8,16 +8,13 @@ import (
...
 	"github.com/ipfs/go-cid"
-	logging "github.com/ipfs/go-log/v2"
 
 	"github.com/filecoin-project/go-address"
 	"github.com/filecoin-project/go-state-types/abi"
 	"github.com/filecoin-project/go-state-types/network"
 	builtin2 "github.com/filecoin-project/specs-actors/v2/actors/builtin"
 )
 
-var log = logging.Logger("buildconstants")
```
Comment (ZenGround0) [inline build/buildconstants/params_interop.go:19]: What about the log call on Line 86, what logger does it call?

**Review by ZenGround0 — COMMENTED** [review]
This seems fine but I don't understand what broke.  Was it something only on the curio side?

File: build/buildconstants/params_testground.go [hunk 2]
```diff
@@ -140,3 +140,8 @@ const BootstrapPeerThreshold = 1
 const Eip155ChainId = 31415926
 
 var MinDealDuration, MaxDealDuration = policy.DealDurationBounds(0)
+
+const UnixfsChunkSize uint64 = 1 << 20
+const UnixfsLinksPerLevel = 1024
+
+var MinVerifiedDealSize = abi.NewStoragePower(256)
```
Comment (rvagg) [inline build/buildconstants/params_testground.go:147]:
```suggestion
```

these were removed in #12217 but restored again in your PR, so just remove here

also build/params_shared_vals.go needs UnixfsChunkSize, UnixfsLinksPerLevel removed and if you `git grep M..DealDuration -- build/` you'll see those vars that were also restored.

File: build/buildconstants/params_interop.go (hunk 1 above)
Comment (rvagg) [reply build/buildconstants/params_interop.go:19]: there's one in build/buildconstants/shared_funcs.go that they all should use, however it should probably be changed from `"build/buildtypes"` to `"build/buildconstants"` @snadrus

**Comment by snadrus** [comment]
resolved

File: build/buildconstants/params_testground.go (hunk 2 above)
Comment (rvagg) [reply build/buildconstants/params_testground.go:147]: Still needs addressing in this file

**Comment by snadrus** [comment]
What broke: 
"make " + ['debug', 'butterflynet', 'calibnet', 'interopnet'] don't build, but no test verifies this until it hits master.

**Comment by rvagg** [comment]
Thanks @snadrus; I agree, and noted in https://github.com/filecoin-project/lotus/issues/12239

**Review by rvagg — APPROVED** [review]

**Pull request merged**